webpack.mix.js

```diff
@@ -0,0 +1,54 @@
+/*
+Copyright the Fluidic Eleventy copyright holders.
+
+See the AUTHORS.md file at the top-level directory of this distribution and at
+https://github.com/fluid-project/fluidic-11ty/raw/master/AUTHORS.md.
+
+Licensed under the New BSD license. You may not use this file except in compliance with this License.
+
+You may obtain a copy of the New BSD License at
+https://github.com/fluid-project/fluidic-11ty/raw/master/LICENSE.md.
+*/
+
+"use strict";
+
+const mix = require("laravel-mix");
+const moveFile = require("move-file");
+const fg = require("fast-glob");
+
+// Set public path.
+mix.setPublicPath("dist/assets");
+
+// Process JavaScript with Babel.
+const scriptPaths = fg.sync("./src/assets/scripts/*.js");
+
+scriptPaths.forEach(entryPath => {
+    mix.js(entryPath, "dist/assets/scripts");
+});
+
+// Process CSS with PostCSS.
+mix.postCss("src/assets/styles/app.css", "dist/assets/styles");
+
+// Compile Sass and process with PostCSS.
+// mix.sass("src/assets/styles/app.scss", "dist/assets/styles");
+
+// Compile Stylus and process with PostCSS.
+// mix.stylus("src/assets/styles/app.styl", "dist/assets/styles");
```

Comment:
I think you may have forgotten to uncomment line 33 and 36

Reply:
I didn't forget—because the input files are all called `app.{css|scss|styl}` it would cause a filename collision if we tried to compile them all to the same output path. The idea here is to show what the configuration would look like for CSS, Sass, and Stylus, not to have active configurations for all three at once.

Reply:
Okay so this will only process a single file called `app.{css|scss|styl}` and by default that is `app.css` If I want instead to use an scss file I'd have to add the scss file and modify this file to process the new scss file instead . 

I don't think this is clear in the documentation, and the automated checking was a bit nicer before. 

If you want to say have your styles separated into separate files, you'd need to import them into `app.{css|scss|styl}` to be processed and usable by the site? Is that correct?

Would it be possible to go through the assets/styles directory and just process each file as needed and output to distribute/assets/styles. I think the only changes you'd need to do would be to loop through the style files and run the appropriate processor/compiler and to use different names for your example style files. In this way the project using this setup can use whichever file name and type they want without having to touch the configuration file. I guess a downside is that if you want separate files but have them imported into a single file, you'll need to add those in a separate directory.

Another thought. What if you want different styles for different pages? For example your news posts vs the landing page.

Reply:
> Okay so this will only process a single file called `app.{css|scss|styl}` and by default that is `app.css` If I want instead to use an scss file I'd have to add the scss file and modify this file to process the new scss file instead .
>
> I don't think this is clear in the documentation, and the automated checking was a bit nicer before.

I included all three file types in `asset/styles`, the idea being that you could delete the ones you don't need and uncomment the configuration line you needed, deleting or commenting out the rest. I've changed things so that the configuration automatically processes all files at the top level of the `src/assets/styles` directory, regardless of file type. This necessitated removing the example Sass and Stylus files to avoid output file conflicts but I've clarified that in the documentation.

> If you want to say have your styles separated into separate files, you'd need to import them into `app.{css|scss|styl}` to be processed and usable by the site? Is that correct?

That's correct. It really depends on how you want to structure your CSS/Sass/Stylus.

> Would it be possible to go through the assets/styles directory and just process each file as needed and output to distribute/assets/styles. I think the only changes you'd need to do would be to loop through the style files and run the appropriate processor/compiler and to use different names for your example style files. In this way the project using this setup can use whichever file name and type they want without having to touch the configuration file. I guess a downside is that if you want separate files but have them imported into a single file, you'll need to add those in a separate directory.

That's not a downside so much as if you want to structure your CSS/Sass/Stylus with imports you just have to do that anyway :) I've made this change so now all the filetypes are handled automatically.

> Another thought. What if you want different styles for different pages? For example your news posts vs the landing page.

Multiple CSS/Sass/Stylus files are supported now so this should "just work". However, the files are still loaded on all pages so I need to look into that.